### ISSUE TITLE
Getting error while opening react devtools in flipper

### ISSUE THREAD

**Issue by nitinshukla413** [facebook/flipper#3933]
When I installed the app and open the flipper it works well but when I open react devtools in flipper, my app got crashes it shows lot of same errors as in the image attached below.

<img width="646" alt="Screenshot 2022-07-25 at 2 39 14 PM" src="https://user-images.githubusercontent.com/50632879/180741181-0a4b3a7c-ed8b-451c-8623-fb5a12b6c265.png">

I also tried to change the flipper version but the issue is still same.

### To reproduce
- install react native app
- yarn install
- cd iOS && pod install && cd..
- start app
- start flipper
- open react devtools in flipper the app got crashed

### Environment
- react-native -0.64.2
- flipper desktop-0.138.0
-System:
    OS: macOS 12.2.1
    CPU: (8) arm64 Apple M1
    Memory: 104.73 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.14.0 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 8.3.1 - /usr/local/bin/npm
    Watchman: Not Found
  Managers:
    CocoaPods: 1.11.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.2, iOS 15.2, macOS 12.1, tvOS 15.2, watchOS 8.3
    Android SDK: Not Found

 

So not sure what the issue is.

Any help is much appreciated.

Thanks

**Comment by LukeDefeo** [comment]
Could you try updating flipper to the latest version

**Comment by nitinshukla413** [comment]
> Could you try updating flipper to the latest version


Yes I try to update the flipper but the issue still remains the same.

**Comment by nitinshukla413** [comment]
> Could you try updating flipper to the latest version

Yes i try to update the flipper but the issue still remains the same

**Comment by jrhager84** [comment]
Same here. Can't use devTools. It will crash the app and throw errors. Updating did nothing the last 3 times I tried but the error remains.
